plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -5055,6 +5069,29 @@ private Answer execute(MigrateVolumeCommand cmd) {
                         volumePath = vmMo.getVmdkFileBaseName(disk);
                     }
             }
+            if (cmd.getNewIops() != null) {
+                String vmwareDocumentation = "https://kb.vmware.com/s/article/68164";
```

Comment:
What if new disk offering does not have IOPS limitation?

Reply:
Then the volume will not have any IOPS limitation.

Reply:
@SadiJr @BryanMLima I think that what @weizhouapache meant is that: if you have a volume using a disk offering with a given IOPS limitation and change the offering to one without IOPS limits, since this code only executes for IOPS != null, the volume will not have its IOPS updated, it will stay with the old value. To me it also looks like this will be the case.

Reply:
@JoaoJandre @weizhouapache, I verified this situation and the IOPS was not updated. I spoke with @SadiJr, and apparently, ACS utilized -1 for offerings with no limitations, but it is not the case anymore. The last commit address this situation.